dev/io.openliberty.io.smallrye.reactive.messaging-provider4/src/io/smallrye/reactive/messaging/providers/extension/AbstractEmitter.java

```diff
@@ -0,0 +1,210 @@
+package io.smallrye.reactive.messaging.providers.extension;
```

Comment:
Need to include an appropriate copyright statement, use what we have put into the EmitterImpl class.

Also need to include a link back to the original source class given this is a chance to that. again EmitterImpl has the details